express/code/scripts/embed-videos.js

```diff
@@ -0,0 +1,73 @@
+import { getLibs } from './utils.js';
+import { getAvailableVimeoSubLang } from './widgets/video.js';
+
+const { createTag, loadScript, loadStyle } = await import(`${getLibs()}/utils/utils.js`);
+
+export function getDefaultEmbed(url) {
+  return `<div style="left: 0; width: 100%; height: 0; position: relative; padding-bottom: 56.25%;">
+    <iframe src="${url.href}" style="border: 0; top: 0; left: 0; width: 100%; height: 100%; position: absolute;" allowfullscreen=""
+      scrolling="no" allow="encrypted-media" title="Content from ${url.hostname}" loading="lazy">
+    </iframe>
+  </div>`;
+}
+
+export function embedYoutube(url) {
```

Comment:
you don't need this @jsandland . Milo should be taking care of a lot of these embeds using autoblocks. Can you remove this whole file?